python/device/device.py

```diff
@@ -942,120 +940,158 @@ def test_reported_properties_threads(self, worker_thread_info):
         can add a property, verify that it was added, then remove it and verify that it was removed.
```

Comment:
It's not letting me comment on the actual function signature line, but it think `test_reported_properties_threads` should be `test_reported_properties_thread` (singular not plural)